xc-admin/packages/xc-admin-common/src/wormhole.ts

```diff
@@ -0,0 +1,10 @@
+import { PythCluster } from "@pythnetwork/client/lib/cluster";
+import { PublicKey } from "@solana/web3.js";
+
+export const WORMHOLE_ADDRESS: Record<PythCluster, PublicKey | undefined> = {
```

Comment:
I suggest adding `localnet` in general. I'm not sure you are using tilt for testing or not but it would be great if as you go add it to the tilt. If you decide to do it later, there is a chance that you never do it 😅

Reply:
I need to add localnet to PythCluster first, which is WIP

Reply:
Done